src/components/shared/FilterNavigatorBar.css

```diff
@@ -125,25 +132,30 @@
 
 /* Animation */
 
+.filterNavigatorBarUncommittedTransition-exit {
+  /* Because of the underlying transition library, this element is still here
+   * while the new "committed" element is created, which pushes it further
+   * right. By using display: none here, we prevent this bad effect. */
+  display: none;
+}
+
 .filterNavigatorBarTransition-enter {
-  z-index: 0;
-  opacity: 0.1;
-  transform: translateX(-100%);
+  color: inherit;
+  opacity: 0.65;
 }
 
 .filterNavigatorBarTransition-enter.filterNavigatorBarTransition-enter-active {
+  color: var(--internal-selected-color);
   opacity: 1;
-  transform: translateX(0);
-  transition: opacity 300ms ease-out, transform 300ms ease-out;
+  transition: color 300ms, opacity 300ms;
 }
 
 .filterNavigatorBarTransition-exit {
   opacity: 1;
-  transform: translateX(0);
 }
 
 .filterNavigatorBarTransition-exit.filterNavigatorBarTransition-exit-active {
-  opacity: 0.01;
+  opacity: 0.1;
   transform: translateX(100%);
   transition: opacity 300ms, transform 300ms;
```

Comment:
These suggestions may be out of scope with this patch, but since we're considering the range selections as a whole, I will give this optional feedback:

*Suggestion:* speed this up to photon's "250ms", as it's not quite snappy enough according to the photon guidelines.

*Suggestion:* The exit transition feels "poppy" as not everything is transitioned.

The exit animation feels a bit "poppy" or "janky" because not everything transitions according to the same time scale.

### Frame analysis:

First state:
<img width="210" alt="image" src="https://user-images.githubusercontent.com/1588648/98601537-cdf07180-22a4-11eb-8f70-6a121328ead1.png">

Frame 1 of animation
Issue: The color of 610ms does not transition.
Issue: The `>` element immediately disappears instead of fading out.
<img width="231" alt="image" src="https://user-images.githubusercontent.com/1588648/98601781-23c51980-22a5-11eb-9098-71456cea2018.png">

Frame N of animation:
The number fades out to the right.
<img width="272" alt="image" src="https://user-images.githubusercontent.com/1588648/98602013-6d156900-22a5-11eb-8d68-e3b79b8fc020.png">

Reply:
I think I fixed most of these issues, except that one:

>  Issue: The > element immediately disappears instead of fading out.

Indeed this is a bit more difficult, because the element itself disappears when the previous "time" item becomes the new leaf item. So this would either need to change a lot of things here, or make some CSS selectors a bit more complex (eg: display the `>` element when the next element is in a transition). I think we should rethink this component a bit more broadly because I believe this predates Flex, and I think we could possibly make it simpler now...

Anyway, more than I want to do here!